pkg/crd/kubermatic/v1/settings.go

```diff
@@ -102,6 +103,20 @@ type MlaOptions struct {
 	MonitoringEnforced bool `json:"monitoringEnforced"`
 }
 
+type ClusterTemplateOptions struct {
+	// Predefined cluster template sizes
+	Sizes []ClusterTemplateSize `json:"clusterTemplateSizes"`
+}
+
+type ClusterTemplateSize struct {
+	// Size name. Can be for example: small, large, L, XL
+	Name string `json:"name"`
+	// Number of clusters
+	NumberOfClusters int64 `json:"numberOfClusters"`
+	// Number od machine deployments
+	NumberOfMachineDeployments int64 `json:"numberOfMachineDeployments"`
```

Comment:
are we talking here about replicas of the same cluster and replicas of the same machine deployments?